src/services/ceramic-service.ts

```diff
@@ -27,8 +27,24 @@ export default class CeramicServiceImpl implements CeramicService {
   }
 
   async loadDocument<T extends Doctype>(docId: DocID): Promise<T> {
+    let timeout: NodeJS.Timeout;
+
     const docPromise = this._client.loadDocument(docId, {sync: false})
-    const timeoutPromise = new Promise((_, reject) => setTimeout(() => reject(`Timed out loading docid: ${docId.toString()}`), 60 * 1000))
+      .then((data) => {
```

Comment:
I think you can use `.finally()` to avoid needing both a `then` and `catch` handler

Reply:
Cool ty!